test/bigtest/tests/uploading-jobs-display-test.js

```diff
@@ -246,4 +246,60 @@ describe('Uploading jobs display', () => {
       });
     });
   });
+
+  describe('when there are marc records exist', () => {
```

Comment:
"when marc records exist" would be better

Reply:
Agree. Changed.